cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -58,6 +61,9 @@ var (
 	argDomain              = flag.String("domain", "cluster.local", "domain under which to create names")
 	argEtcdMutationTimeout = flag.Duration("etcd-mutation-timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
 	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
+	argEtcdCAFile          = flag.String("etcd-cafile", "", "Certificate Authority file to secure etcd communication")
+	argEtcdCertFile        = flag.String("etcd-certfile", "", "TLS certificate file to secure etcd communication")
+	argEtcdKeyFile         = flag.String("etcd-keyfile", "", "TLS key file to secure etcd communication")
```

Comment:
I mostly meant the implementation might save you work. But also, the
command line flag names & help strings should be consistent.

On Tue, Mar 22, 2016 at 1:07 PM, a-palchikov <notifications@github.com>
wrote:

> In cluster/addons/dns/kube2sky/kube2sky.go
> <https://github.com/kubernetes/kubernetes/pull/23331#discussion_r57059306>
> :
>
> > @@ -58,6 +61,9 @@ var (
> >  	argDomain              = flag.String("domain", "cluster.local", "domain under which to create names")
> >  	argEtcdMutationTimeout = flag.Duration("etcd-mutation-timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
> >  	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
> > +	argEtcdCAFile          = flag.String("etcd-cafile", "", "Certificate Authority file to secure etcd communication")
> > +	argEtcdCertFile        = flag.String("etcd-certfile", "", "TLS certificate file to secure etcd communication")
> > +	argEtcdKeyFile         = flag.String("etcd-keyfile", "", "TLS key file to secure etcd communication")
>
> @lavalamp <https://github.com/lavalamp> By being consistent here you mean
> to support the rest of EtcdConfig like Prefix and Quorum?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23331/files/48e7e271b7b4463daba3bdd90b2ea7a029e2d9a7#r57059306>
>